src/operation-node/drop-type-node.ts

```diff
@@ -2,12 +2,14 @@ import { freeze } from '../util/object-utils.js'
 import { OperationNode } from './operation-node.js'
 import { SchemableIdentifierNode } from './schemable-identifier-node.js'
 
-export type DropTypeNodeParams = Omit<Partial<DropTypeNode>, 'kind' | 'name'>
+export type DropTypeNodeParams = Omit<Partial<DropTypeNode>, 'kind' | 'names'>
```

Comment:
This type is used in the `cloneWith` below. We don't want `name` and `names` to be overriden after `create`.

```suggestion
export type DropTypeNodeParams = Omit<Partial<DropTypeNode>, 'kind' | 'name' | 'names'>
```